packages/@vuepress/markdown/src/plugins/codePlugin/resolveLanguage.ts

```diff
@@ -32,7 +32,7 @@ const getLanguagesMap = (): LanguagesMap => {
  */
 export const resolveLanguage = (info: string): HighlightLanguage => {
   // get user-defined language alias
-  const alias = info.match(/^([a-zA-Z]+)/)?.[1] || 'text'
+  const alias = info.match(/^([a-zA-Z-]+)/)?.[1] || 'text'
```

Comment:
The dash should not be the first char 🤔

Reply:
Yes, but I think it's not really necessary to perform a strict check.

Maybe vite's idea, fast should be the first choise?

Reply:
It's not for check. The alias match rule should be correct.